### PR TITLE
Solution for issue #311

### PR DESCRIPTION
**Pull request by isaacfi**
To solve the issue [Issue 311](https://github.com/rochacbruno/flasgger/issues/311) I loaded the main_def from file using $ref property and absolute application path

### CODE REVIEW

**Comment by isaacfi** [comment]
I think that is a mistake while paste the code, let me delete the blank line

**Comment by isaacfi** [comment]
Please accept my solution: @rochacbruno

File: flasgger/utils.py [hunk 1]
```diff
@@ -251,6 +252,36 @@ def wrapper(*args, **kwargs):
     return decorator
 
 
+def __replace_ref(schema: dict, relative_path: str):
+    absolute_path = os.path.dirname(sys.argv[0])
+    new_value = {}
+    for key, value in schema.items():
+        print('key:', key)
```
Comment (billyrrr) [inline flasgger/utils.py]: @isaacfi Could you delete the print statements? Thanks

**Comment by billyrrr** [comment]
Hi, @isaacfi . I have merged your commit into the master branch in this [commit](https://github.com/rochacbruno-archive/flasgger/commit/5ee9586febd931cbdce54527f40c53def4d747d1) Please let me know if you have any questions. Thank you for your contribution.

File: flasgger/utils.py (hunk 1 above)
Comment (isaacfi) [reply flasgger/utils.py]: Yes of course I forgot to delete them

**Pull request closed without merging**